.circleci/config.yml

```diff
@@ -762,6 +790,34 @@ jobs:
                 variant: "debug"
       - run: exit 0
 
+  billing-tests:
+    executor: ndk-r22-latest-executor
+    environment:
+      JVM_OPTS: -Xmx3200m
+      BUILDTYPE: Debug
+      GRADLE_OPTS: -Xmx4096m -Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process
+    steps:
+      - checkout
+      - when:
+          condition:
+            not:
+              matches:
+                pattern: "^add-changelog.*"
+                value: << pipeline.git.branch >>
```

Comment:
Do I understand correctly, that if a few commits are added during the day, but the latest one is changelog change, we won't run tests, will we?

Reply:
We won't run them anyway for any commit, because `billing-tests` is not part of `default` workflow. These tests will run only on a schedule https://github.com/mapbox/mapbox-navigation-android/pull/7042/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R71

So we don't really need this condition here. I left this condition just in case if someday `billing-tests` will run on other schedule and we don't miss changelog condition. But it's arguable whether this condition should be here now.